src/styles/_exchange-form.scss

```diff
@@ -191,3 +191,20 @@
   color: white;
   font-size: 14px;
 }
+
+.account-select {
+  .v-select__selection {
+    font-size: 15px;
+  }
+}
+
+.amount-input {
+  font-size: 15px;
+}
+
+.wallet-address-input {
+  font-size: 11px;
```

Comment:
why is the size of the address smaller than the account and amount? The three of them should have the same size, I even suggest using the same class for this

Reply:
Because wallet address haven't space in to the label and the others labels see very small with 11px